ajax/libs/sketchpad/package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "sketchpad",
+  "version": "0.1.0",
+  "description": "A simple sketchpad.",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/yiom/sketchpad.git"
+  },
+  "homepage": "https://yiom.github.io/sketchpad/",
+  "keywords": [
+    "sketchpad",
+    "javascript"
+  ],
+  "author": "Yiom <https://yiom.com.br>",
```

Comment:
I can't access this website https://yiom.com.br
Can you?
If you also can't access this website, let's remove it.
What do you think?

Thank you.